### PR TITLE
Update react package name and add homepage and bugs section

### PR DESCRIPTION
**Pull request by louislingatong**

### CODE REVIEW

File: packages/stencil/src/components/datacom-tabs/datacom-tabgroup.css
Comment (4leite) [inline packages/stencil/src/components/datacom-tabs/datacom-tabgroup.css:1]: is this meant to be in here?

File: packages/stencil/src/components/datacom-tabs/datacom-tabgroup.tsx
Comment (4leite) [inline packages/stencil/src/components/datacom-tabs/datacom-tabgroup.tsx:1]: as above

File: packages/stencil/src/components/datacom-tabs/datacom-tabgroup.css
Comment (louislingatong) [reply packages/stencil/src/components/datacom-tabs/datacom-tabgroup.css:1]: Oops! it shouldn't be here. will revert this change.

File: packages/stencil/src/components/datacom-tabs/datacom-tabgroup.tsx
Comment (louislingatong) [reply packages/stencil/src/components/datacom-tabs/datacom-tabgroup.tsx:1]: Oops! it shouldn't be here. will revert this change

**Pull request merged**